.github/workflows/unit-tests.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Set up python
         uses: actions/setup-python@v4
```

Comment:
While you're at it, could you update this version pin here to the latest version, v5, as well?